load-test-framework/src/main/java/com/google/pubsub/flic/Driver.java

```diff
@@ -372,6 +379,11 @@ public void run(BiFunction<String, Map<ClientParams, Integer>, Controller> contr
           broker != null || (kafkaPublisherCount == 0 && kafkaSubscriberCount == 0),
           "If using Kafka you must provide the network address of your broker using the"
               + "--broker flag.");
+      Preconditions.checkArgument(
```

Comment:
This depends - if no Zookeeper is provided, then the topic deletion/creation will be optional.